shielddash/core/static/js/app/api/study-api.js

```diff
@@ -31,9 +31,9 @@ export function getStudies() {
   return axios.get(endpoints.GET_STUDIES, {headers}).then(response => {
     store.dispatch(getStudiesSuccess(response.data));
     return response;
-  }).catch(err => {
-    console.error(err);
-    store.dispatch(getStudiesFailure(err.status));
+  }).catch(response => {
```

Comment:
To clarify, I think `err` is also just a response, according to https://github.com/mzabriskie/axios#response-schema Is that correct?